Triphouse-project/index.html

```diff
@@ -18,26 +18,47 @@
             <div class="wrapper_desktop">
```

Comment:
![2021-04-22_22h07_27](https://user-images.githubusercontent.com/19935417/115771975-419bbc00-a3b7-11eb-994f-cc11241e258f.png)
Align content on the top block.